pages/contact.html

```diff
@@ -0,0 +1,247 @@
+              <input
+                type="text"
+                placeholder="Full Name"
+                class="w-full h-16 px-5 rounded-lg font-roboto outline-none"
+              />
+              <input
+                type="text"
+                placeholder="Company Name"
+                class="w-full h-16 px-5 rounded-lg font-roboto outline-none"
+              />
+            </div>
+            <input
+              type="email"
+              placeholder="work Email"
+              class="w-full h-16 px-5 rounded-lg font-roboto outline-none"
+            />
+            <input
+              type="text"
+              placeholder="Subject"
+              class="w-full h-16 px-5 rounded-lg font-roboto outline-none"
+            />
+            <textarea
+              placeholder="message"
+              class="font-roboto p-5 max-h-36 h-36 rounded-lg resize-none outline-none"
+            ></textarea>
+          </form>
+          <button class="primary-btn--full-width">send</button>
+        </div>
+        <div class="flex flex-col w-fit mx-auto mt-20 gap-5">
+          <div class="flex items-center gap-6">
+            <img src="../assets/vectors/phone-icon.svg" />
+            <div class="flex flex-col items-start">
+              <h4>+ 1800 145 276</h4>
+              <p class="text-primary/80">Free support</p>
+            </div>
+          </div>
+          <div class="flex items-center gap-6">
+            <img src="../assets/vectors/send-icon.svg" />
+            <div class="flex flex-col items-start">
+              <h4>finsweet@gmail.com</h4>
+              <p class="text-primary/80">Help Email support</p>
+            </div>
+          </div>
+          <div class="flex items-center gap-6">
+            <img src="../assets/vectors/email.svg" />
+            <div class="flex flex-col items-start">
+              <h4>sales@finsweet.com</h4>
+              <p class="text-primary/80">Sales Enquiry</p>
+            </div>
+          </div>
+        </div>
+      </section>
+    </section>
+
+    <section class="flex flex-col mt-20">
+        <div
+          class="flex flex-col justify-center items-center gap-9 relative pb-32"
+        >
+          <h1 class="text-center leading-[60px]">
+            Are you ready to grow your<br />
+            business with us?
+          </h1>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+          <button class="primary-btn">View Pricing</button>
+  
+          <img
+            src="../assets/images/BlurGradient-footer.png"
+            class="absolute bottom-0 w-full h-full -z-10 blur-3xl -mb-16"
+          />
+        </div>
+  
+        <footer class="bg-primary text-gray-400 py-20 px-20 flex flex-col gap-9">
+          <div class="font-roboto grid grid-cols-5">
+            <ul class="flex flex-col gap-3">
+              <li class="text-white font-bold paragraph-large">Company</li>
```

Comment:
I suggest paragraph-large to be a modifier, I think all paragraphs should have the same class and then use paragraph large as a modifier to modify a paragraph the way you want.